src/create_disk.sh

```diff
@@ -172,13 +172,17 @@ case "$arch" in
         ;;
     s390x)
         sgdisk_args=()
+        rootp_end=0
         if [[ ${secure_execution} -eq 1 ]]; then
             # shellcheck disable=SC2206
             sgdisk_args+=(-n ${SDPART}:0:+200M -c ${SDPART}:se -t ${SDPART}:0FC63DAF-8483-4772-8E79-3D69D8477DE4)
+            # we need to leave space for the verity hash partitions
+            rootp_end=-$((128+256+1))M
```

Comment:
Where does the +1 come from?

Reply:
I still don't know where the +1 comes from.

Reply:
I didn't go deep in the accounting here, but for some reason leaving 384M free at the end of the disk is not enough to fit exactly two partitions that sum up to 384M. I added a comment at least.